src/signals/incident/components/form/MapSelectors/Caterpillar/CaterpillarLayer/CaterpillarLayer.tsx

```diff
@@ -98,24 +104,42 @@ export const CaterpillarLayer: FC = () => {
   const reportedFeatureType = meta.featureTypes.find(
     ({ typeValue }) => typeValue === 'reported'
   ) as FeatureType
-  const reportedFeatures = features.filter((feature) =>
-    Boolean(
-      reportedFeatureType?.isReportedField &&
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        feature.properties![reportedFeatureType.isReportedField] ===
-          reportedFeatureType.isReportedValue
-    )
+  const checkedFeatureType = meta.featureTypes.find(
+    ({ typeValue }) => typeValue === 'checked'
+  ) as FeatureType
+
+  const statusFeatures = features.filter(
+    (feature) =>
+      Boolean(
+        reportedFeatureType.isReportedField &&
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          feature.properties![reportedFeatureType.isReportedField] ===
+            reportedFeatureType.isReportedValue
+      ) ||
+      Boolean(
+        checkedFeatureType.isCheckedField &&
+          checkedFeatureType.isCheckedValues?.includes(
+            // eslint-disable-next-line
+            // @ts-ignore
```

Comment:
Why should this line be ignored?

Reply:
this is fixed now with the no-non-null-assertion line